docs/usage.rst

```diff
@@ -40,10 +40,10 @@ parameter for emissions. Thus, it is identified by
 
     ``Emissions`` -> ``CO2`` -> ``Industrial``.
 
-In the low-level API parameters are expected to be identified by
-tuples of strings describing their position in the hierarchy, i.e. in
-this example ``("Emissions", "CO2", "Industrial")``. The high-level
-API also allows for giving it as a string with the hierarchical levels
+In the core API parameters are expected to be identified by tuples of
+strings describing their position in the hierarchy, i.e. in this
+example ``("Emissions", "CO2", "Industrial")``. The ScmDataFrame API
```

Comment:
```suggestion
example ``("Emissions", "CO2", "Industrial")``. 
```

Delete the whole ScmDataFrame API sentence. We'll explain how to access data when we write the ScmDataFrame API docs